### ISSUE TITLE
Provisioning files or running provision shell don't modify docker container files

### ISSUE THREAD

**Issue by laurilarjo** [Seravo/wordpress#92]
My goal:
Trying to modify xdebug.ini -settings to enable debugging with VSCode. The file is located in folder "/etc/php/mods-available".

I tried to do this by uploading a correct xdebug.ini-file to home-dir, and then moving it to the correct location. So I started by modifying Vagrantfile and adding
`config.vm.provision "file", source: "./vagrant-files/xdebug.ini", destination: "~/xdebug.ini"`
I verified this worked by writing a new line right after:
`config.vm.provision "shell", inline: "ls -la /home/vagrant/"` and this displayed my file was there.

However, when I then `vagrant ssh` to the box, I don't see this file. I figured it's because when I `vagrant ssh` to the box, I end up in a docker container, whereas the provisioning commands in Vagrantfile modify the vagrant box. Also when I run on host machine `vagrant ssh -c "ls -la /home/vagrant"` I see the uploaded xdebug.ini file, as I'm not in the docker container.

How could I upload files or run commands in the docker container, as I need to edit the /etc/php/mods-available/xdebug.ini? This file or folder doesn't even exist in vagrantbox.

**Comment by ottok** [comment]
What is the content of your custom xdebug.ini? What is the goal you are trying to achieve? Start the interactive debugger or something else?

**Comment by laurilarjo** [comment]
Yes, starting the interactive debugger, and for this I needed to set 
- xdebug.remote_autostart = 1
- xdebug.remote_connect_back = 1

**Comment by ottok** [comment]
Hello!

Sorry for the delay in follow-up.

There was bugs in Xdebug that prevented you from getting it working even if you would have the remote_connect_back = 1. Those were fixed in release 20190122.0.0 (https://seravo.com/docs/get-started/release-notes/#vagrant-box-seravowordpress-2019012200).

Now by default after running `wp-xdebug-on` the settings look like this:
```
vagrant@development:~$ cat /etc/php/7.2/fpm/conf.d/20-xdebug.ini 
; Load XDebug
zend_extension=xdebug.so

; Enable PHP profiling with get param XDEBUG_PROFILE=1
xdebug.profiler_output_dir=/tmp
xdebug.profiler_output_name=cachegrind.out.%t.%p
xdebug.profiler_enable_trigger=1

; Enable PHP tracing with get param XDEBUG_TRACE=1
xdebug.trace_enable_trigger=1
xdebug.trace_format=1
xdebug.show_mem_delta=1

; Enable dumping detailed info in the frontend when PHP fatals
xdebug.collect_vars=on
xdebug.collect_params=4
xdebug.dump_globals=on
xdebug.show_local_vars=on
xdebug.dump.SERVER=HTTP_HOST
xdebug.dump.COOKIE=*
xdebug.dump.POST=*

; Enable php debugging with idekey 'vagrant'
xdebug.default_enable=1
xdebug.idekey="vagrant"
xdebug.remote_enable=1
xdebug.remote_autostart=0
xdebug.remote_port=9000
xdebug.remote_handler=dbgp
xdebug.remote_log="/data/log/xdebug.log"
; Allow any IP, see details at https://xdebug.org/docs/remote
xdebug.remote_connect_back=1
```

You shouldn't need `autostart`, it will work out-of-the-box without it as well.

Our docs on Xdebug were also updated in https://github.com/Seravo/docs/commit/3acea49803aac6ae1e5447b4c090a9106a3822e9 (https://seravo.com/docs/development/xdebug/)

With this I believe the root cause of your problems is fixed and you can now use Xdebug again. Feel free to repoen this issue if this didn't solve it for you.
